test/src/features/MapsScreen/screens/MapsHome/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from "react";
```

Comment:
The parent folder of this file should be a conditional rendering folder, right now is MapsHome :)

Reply:
What do you mean by this?